README.md

```diff
@@ -1,2 +1,73 @@
 # cs-entities
 Simple CloudStack Entities Framework
+
+The library provides the convenient way of working with Apache CloudStack entities through the following mechanisms:
+1. Creating and retrieving the declared entities on/from Apache CloudStack server, such as users, accounts, virtual machines, tags.
+It based on extensible Request which simplify creating requests to CloudStack.
+2. Base set of Apache CloudStack events, for retrieving their from Apache CloudStack messages.
+
+## Install with SBT
+
+Add the following to your `build.sbt`
+```scala
+libraryDependencies += "com.bwsw" %% "cs-entities" % "4.9.3"
+```
+## Getting Started      
+
+The diagram below is a simple illustration of how the library's classes should be used. \
```

Comment:
It isn't true because it illustrates a work of DAO and what's more - the last sentence doesn't describe a general usage of our library and a special case